src/item.cpp

```diff
@@ -4449,7 +4558,7 @@ std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int t
         }
     }
 
-    const sizing sizing_level = get_sizing( g->u, get_encumber( g->u ) != 0 );
+    const sizing sizing_level = get_sizing( g->u );
```

Comment:
Now I've seen a few examples of this I think a simple solution would be to replace all the places doing `get_encumber( g->u ) != 0` with a new `has_any_encumbrance` function (or your choice of name).  Then you wouldn't need to change `get_sizing`.

Reply:
Why though? The returning enum value is never referenced in code. It's dead. It should be removed imo.

Reply:
If this instance isn't used then indeed it should be removed, but there were others.

Reply:
I don't understand what you mean. This passed bool was only used to return the enum value sizing::not_wearable. If the passed bool was false, get_sizing did nothing except return this value. A value which is not used anywhere for anything from what I can see.

Reply:
OK, I agree that the implementation of `get_sizing` and the name of that parameter were poor.  But the point of the enum value that was returned in this case was to be the "encubrance doesn't matter" value, which is used, in an indirect way; it's the value that leads to nothing being printed.  We don't want to print things like `(too big)` when there's zero encumbrance, because sizing is irrelevant for such items.  So I think `get_sizing` should still return this value when that's the case.

However, you're right that this could be done more cleanly. The extra argument to `get_sizing` isn't needed; this determination could be made inside `get_sizing` rather than needing to be an extra argument.

Reply:
I have modified get_sizing so that it should now function as before. Let me know what you think.